src/EditorFeatures/CSharpTest/PdbSourceDocument/PdbFileLocatorServiceTests.cs

```diff
@@ -55,7 +55,9 @@ public class C
             {
                 MarkupTestFile.GetSpan(source, out var metadataSource, out var expectedSpan);
 
-                var (project, symbol) = await CompileAndFindSymbolAsync(path, Location.OnDisk, Location.OnDisk, metadataSource, c => c.GetMember("C.E"));
+                // Ideally we don't want to pass in true for windowsPdb here, and this is supposed to test that the service ignores non-portable PDBs when the debugger
+                // tells us they're not portable, but the debugger has a bug at the moment.
+                var (project, symbol) = await CompileAndFindSymbolAsync(path, Location.OnDisk, Location.OnDisk, metadataSource, c => c.GetMember("C.E"), windowsPdb: true);
```

Comment:
Is there a tracking bug for this?

Reply:
Not yet, waiting to hear back from the debugger team. I might just be interpreting the API incorrectly.